lib/devices/Roborock.js

```diff
@@ -5,7 +5,7 @@ const Tools = require("../Tools");
 const RRMapParser = require("../RRMapParser");
 
 /**
- * @implements {MiioVacuum}
+ * @extends {MiioVacuum}
```

Comment:
Same here. Maybe we should start early to have a nev device.js - the viomi is not a Roborock.
Maybe have a "device-independent", "Roborock", "MiioVacuum"?

Reply:
MiioVacuum is device-independent.
Viomi and Roborock are its implementations. The goal is that both conform to the same interface (not yet 100% there).